.changeset/curly-wombats-wink.md

```diff
@@ -0,0 +1,14 @@
+---
+"@ignored/hardhat-vnext-errors": patch
+"@ignored/hardhat-vnext-utils": patch
+"@ignored/hardhat-vnext": patch
+"@ignored/hardhat-vnext-build-system": patch
+"@ignored/hardhat-vnext-keystore": patch
+"@ignored/hardhat-vnext-mocha-test-runner": patch
+"@ignored/hardhat-vnext-network-helpers": patch
+"@ignored/hardhat-vnext-node-test-reporter": patch
+"@ignored/hardhat-vnext-node-test-runner": patch
+"@ignored/hardhat-vnext-zod-utils": patch
```

Comment:
aren't we missing `hardhat-ethers`?

Reply:
I have included `hardhat-ethers` now.
I also changed the version bump to `major` for any packages that have not been previously released so that their version show: `3.0.0-next-0` with this release.